GPU/GLES/ShaderManagerGLES.cpp

```diff
@@ -586,6 +587,31 @@ void LinkedShader::UpdateUniforms(u32 vertType, const VShaderID &vsid) {
 	if (dirty & DIRTY_STENCILREPLACEVALUE) {
 		glUniform1f(u_stencilReplaceValue, (float)gstate.getStencilTestRef() * (1.0f / 255.0f));
 	}
+
+	if ((dirty & DIRTY_TEXLOD) && gstate_c.Supports(GPU_SUPPORTS_EXPLICIT_LOD) && u_texlod) {
+		float lod = 0.0;
+		switch (gstate.getTexLevelMode()) {
+		case GE_TEXLEVEL_MODE_CONST: {
+			float scaleLog = TexLog2F((float)gstate_c.curRTScale);
```

Comment:
So this assumes that all mip levels contain the same image, just scaled, right?  I guess we haven't heard it manually detected by anyone, so maybe it's true.  Unless some game accidentally specifies black or something dumb for some level that never happened so was never noticed.

-[Unknown]